app/views/person/sac_remarks/index.html.haml

```diff
@@ -0,0 +1,20 @@
+-#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+-#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+-#  or later. See the COPYING file at the top-level directory or at
+-#  https://github.com/hitobito/hitobito_sac_cas.
+
+.row
```

Comment:
ich würde hier einfach über die verfügbaren attribute welche du in einer collection im controller definierst iterieren. so musst du die Deklaration welche attrs hier gerender werden/schreibar sind nur an einer zentralen Stelle definieren

Reply:
ich habe das jetzt so versucht, aber es ist deutlich komplizierter, weshalb ich es doch wieder reverted habe